Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -206,12 +206,22 @@ void Settings::SetStateSlot(int slot)
 
 bool Settings::GetRenderToMain() const
 {
-  return value(QStringLiteral("Graphics/RenderToMain"), false).toBool();
+  return SConfig::GetInstance().bRenderToMain;
+}
+
+void Settings::SetRenderToMain(bool rendertomain)
```

Comment:
This doesn't make sense to me. What problem are you trying to solve? If a configuration variable name is changed, a `find | sed -i s/bRenderToMain/myNewName/g` should fix DolphinQt2 just as well as the rest of Dolphin.